### ISSUE TITLE
Extend paramiko client to add custom methods.

### ISSUE THREAD

**Issue by rochacbruno** [SatelliteQE/robottelo#3694]
Currently we can do

``` console
with ssh.get_connection(..) as connection:
    ssh.execute_command('cp /orign /destiny', connection)
    ssh.execute_command('chmod 0777 /destiny', connection)
    ssh.execute_command("echo 'foo' > /destiny/bar", connection)
```

we should extend SSHClient, adding execute_command method (as a proxy) to allow something like:

``` console
with ssh.get_connection(..) as connection:
    connection.execute_command('cp /orign /destiny')
    connection.execute_command('chmod 0777 /destiny')
    connection.execute_command("echo 'foo' > /destiny/bar")
```

**Comment by elyezer** [comment]
:+1:

**Comment by rplevka** [comment]
@rochacbruno what would be the gain?

**Comment by rochacbruno** [comment]
> @rochacbruno what would be the gain?

aesthetic, code style, readability

**Comment by rochacbruno** [comment]
Closed in #3943
